src/scripting/game_lua_kernel.cpp

```diff
@@ -4726,7 +4726,7 @@ void game_lua_kernel::set_game_display(game_display * gd) {
  * elsewhere (in the C++ code).
  * Any child tags not in this list will be passed to Lua's on_load event.
  */
-static const std::array<std::string, 24> handled_file_tags {{
+static const std::array handled_file_tags {
```

Comment:
Same thing again.

(Aside: It seems to me that the function using this could just be an `std::find` call.)